### ISSUE TITLE
Plugin 'SocketIOClient' failed to load because module 'SocketIOClient' could not be found.  Please ensure the plugin is properly installed, otherwise consider disabling the plugin for this project.

### ISSUE THREAD

**Issue by MarkMinerov** [getnamo/SocketIOClient-Unreal#397]
Hello, I have an issue with build. When I install the plugin on an empty project using Unreal Engine 5.3 and try to build for Windows, this is what I get: Plugin 'SocketIOClient' failed to load because module 'SocketIOClient' could not be found.  Please ensure the plugin is properly installed, otherwise consider disabling the plugin for this project. 

What am I doing wrong? I follow the installation step by step:
Download last release for UE 5.3
Add the plugin to Plugins folder in my project
I open plugins menu in UE and check the plugin so my project uses it

Then I try to build, build goes well, but when I start the application (.exe) I get this error, what is wrong? What should I do?
I came here from UDP-Wrapper plugin, which also doesn't work because of SocketIOClient. TCP-Wrapper works well... I think something is wrong with this plugin or I just don't know how to install it properly. Thank you in advance!

**Comment by getnamo** [comment]
Are you using the [SocketIOClient-v2.7.0-UE5.3.7z](https://github.com/getnamo/SocketIOClient-Unreal/releases/download/v2.7.0/SocketIOClient-v2.7.0-UE5.3.7z) link when adding the plugin to project?

**Comment by getnamo** [comment]
Try latest release: https://github.com/getnamo/SocketIOClient-Unreal/releases/tag/v2.7.1. It should work out of the box for 5.3.

**Comment by MarkMinerov** [comment]
Hello, sorry it took me so long to respond... And yes - I use last releases, I installed all last releases of Visual Code, last versions of SDK, I have Windows 11. UE 5.3, etc. 

I just tried your fix, and it still doesn't work. The plugin works when I start the game in "Selected viewport" mode inside UE, but not when I try to build it. **I use Windows 11 to build the game**. The game compiles fine, but when I open it, i see black screen and 
Windows dialog window with message saying: "Plugin 'SocketIOClient' failed to load because module 'SocketIOClient' could not be found. Please ensure the plugin is properly installed, otherwise consider disabling the plugin for this project."

Plugin UDP-Wrapper works well.

**Comment by getnamo** [comment]
Ahh I think I know what you're missing. You need to add dummy c++ code to your project to convert it into a hybrid project so it will include and build the plugin along with it when packaging, see https://github.com/getnamo/SocketIOClient-Unreal#github

**Comment by MarkMinerov** [comment]
Yes, it helped. Thank you very much!

Just for your information:
1. I used `git clone https://github.com/getnamo/SocketIOClient-Unreal.git --recurse-submodules` to clone the plugin in Plugins folder.
2. I opened my project, and it asked me to rebuild the plugin, because some binaries of the libraries it uses cannot be found.
3. I clicked on `Rebuild Now` and just waited for it to rebuild. I think I did the same thing before, I just didn't know if the rebuild is done, or it is still in progress, as Unreal Engine doesn't log if the rebuild is fine or something went wrong.
